pawar-2018/pawar2018
# Mu-Plugins and Events CPT

# What does this pull request do?

It puts plugins into a "mu-plugins" (must use plugins) directory. Plugins in this folder are automatically installed and activated in the WordPress install. And they do not show up in the Plugins section of the Admin.

It also created the Events CPT(custom post type). Events can now be added/edited in wp-admin.

# How should the reviewer test this pull request?

After this is merged into staging and pull in the content I made locally into staging you can see how it works.

ACF plugin should be deleted from Plugins section after this is merged in. 

# Include the Basecamp todo for this PR, if there is one.

https://3.basecamp.com/3666950/buckets/2763013/todos/407240415

- Though homepage events still need to pull from events cpt.

# Include any screenshots that will help explain this PR.

n/a

# Any other context you want to provide?

nah.
